public/index.html

```diff
@@ -4,6 +4,7 @@
     <meta charset="utf-8" />
     <meta name="color-scheme" content="dark light">
     <link rel="shortcut icon" href="%PUBLIC_URL%/favicon.ico" />
+     <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.1/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-+0n0xVW2eSR5OomGNYDnhzAbDsOXxcvSN1TPprVMTNDbiYZCxYbOOl7+AMvyTG2x" crossorigin="anonymous">
```

Comment:
We are not using bootstrap. Downloading another css library may cause a lot of unusual loading time.
We are using material-ui.

Reply:
ok